packages/arb-token-bridge-ui/synpress.config.ts

```diff
@@ -49,9 +54,18 @@ export default defineConfig({
       if (!arbRpcUrl) {
         throw new Error('NEXT_PUBLIC_LOCAL_ARBITRUM_RPC_URL variable missing.')
       }
+      if (!goerliRpcUrl) {
+        throw new Error(
+          'process.env.NEXT_PUBLIC_GOERLI_RPC_URL variable missing.'
```

Comment:
we already provide the infura key in the `.env` file
i don't think the goerli rpc url should be provided as it can be created using the infura key